cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -252,6 +252,10 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
       if ( selectionIndex > -1 ) {
         _selections.splice( selectionIndex, 1 );
 
+        if ( _list.classList.contains( 'max-selections' ) ) {
```

Comment:
You could just remove the class without the `contains` check. `remove()` shouldn't throw an error if the class is not found.